### PR TITLE
[PRM-1276] - Updated API controller for new 1812 API

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

**Comment by LewisMitchell** [comment]
Waiting on #119  and #118

File: app/controllers/APIController.scala [hunk 1]
```diff
@@ -80,4 +111,29 @@ class APIController @Inject()(etmpService: ETMPService,
...
+    val crystallisedPenaltyTotal = apiService.getCrystallisedPenaltyTotal(penaltyDetails)
+    val hasAnyPenaltyData = apiService.checkIfHasAnyPenaltyData(penaltyDetails)
+    val responseData: APIModel = APIModel(
+      noOfPoints = pointsTotal,
+      noOfEstimatedPenalties = noOfEstimatedPenalties,
+      noOfCrystalisedPenalties = crystallisedPenaltyAmount,
+      estimatedPenaltyAmount = penaltyAmountWithEstimateStatus,
+      crystalisedPenaltyAmountDue = crystallisedPenaltyTotal,
+      hasAnyPenaltyData = hasAnyPenaltyData)
+    if(pointsTotal > 0) {
```
Comment (AndrewRButcher) [inline app/controllers/APIController.scala:129]: Not one to fix now *unless you want too) but what about if the user has no LSP and only LPP the audit event would not fire.
Looks like this was the same on previous implementation so happy to raise bug if not fixed now

Comment (LewisMitchell) [reply app/controllers/APIController.scala:129]: Will look into it now

Comment (LewisMitchell) [reply app/controllers/APIController.scala:129]: Will change to `hasAnyPenaltyData`
```suggestion
    if(hasAnyPenaltyData) {
```

Comment (AndrewRButcher) [reply app/controllers/APIController.scala:129]: 👍  Nice yes that is the simplest solution I was thinking we need a few ors in the if statement but yes hasAnyPenaltyData should do what we need

**Review by AndrewRButcher — APPROVED** [review]

**Pull request merged**